setup.cfg

```diff
@@ -34,7 +34,7 @@ test_require =
 [options.extras_require]
 optional =
     pandas
-    cf-units
+    cf-units >= 3.3.0 # Earliest release that works with numpy >=2.0.0
```

Comment:
From the release notes: https://github.com/SciTools/cf-units/releases
* now compatible with numpy v2 (https://github.com/SciTools/cf-units/pull/488)
  *  note : backwards-compatibility with v1 is expected, but no longer tested

Is there a way to say that we require >= 3.3.0 if we have numpy >= 2.0.0 (similar to python_version)? (I don't think so: https://peps.python.org/pep-0508/#environment-markers )

I would make that:
```cf-units # >= 3.3.0 - Earliest release that works with numpy >=2.0.0```
The cf-units problem with numpy is not a problem which pyaro should fix.